From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Ayotte <martinayotte@yahoo.ca>
Date: Thu, 3 Jan 2019 11:04:17 -0500
Subject: [ARCHEOLOGY] add better strength on SDCard and put back previous
 speed setting

> X-Git-Archeology: - Revision 211c802f7cfedaac75a35f65d427910011c29242: https://github.com/armbian/build/commit/211c802f7cfedaac75a35f65d427910011c29242
> X-Git-Archeology:   Date: Thu, 03 Jan 2019 11:04:17 -0500
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: add better strength on SDCard and put back previous speed setting
> X-Git-Archeology:
> X-Git-Archeology: - Revision f2e71dbc730eaf7f1e586dfd5951bc149cb827d2: https://github.com/armbian/build/commit/f2e71dbc730eaf7f1e586dfd5951bc149cb827d2
> X-Git-Archeology:   Date: Fri, 04 Jan 2019 10:04:19 -0500
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: fix sdmmc-bus1 pin
> X-Git-Archeology:
> X-Git-Archeology: - Revision 005e8cf9fcba2922685f5ee49cef833be1465ec1: https://github.com/armbian/build/commit/005e8cf9fcba2922685f5ee49cef833be1465ec1
> X-Git-Archeology:   Date: Sun, 27 Jan 2019 15:37:40 -0500
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: fxi SD-CD pullup
> X-Git-Archeology:
> X-Git-Archeology: - Revision 885111ab0e166f97338df63714ea47a5b554b698: https://github.com/armbian/build/commit/885111ab0e166f97338df63714ea47a5b554b698
> X-Git-Archeology:   Date: Sat, 13 Jul 2019 11:25:27 -0400
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: switch rockchip64-dev and rk3399-dev to 5.2.y
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 4d4c3f58ffc1cbfbb060cbabc9eb414036a2fda5: https://github.com/armbian/build/commit/4d4c3f58ffc1cbfbb060cbabc9eb414036a2fda5
> X-Git-Archeology:   Date: Wed, 02 Sep 2020 23:22:09 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64 curent to kernel 5.8.y (#2175)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 6b490e16944b30ff69bf9c13678905187df0d9d4: https://github.com/armbian/build/commit/6b490e16944b30ff69bf9c13678905187df0d9d4
> X-Git-Archeology:   Date: Tue, 11 Jan 2022 15:26:11 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel edge to 5.16 (#3387)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f52a4193d02ef88333ba117c68d49486dfd7ff41: https://github.com/armbian/build/commit/f52a4193d02ef88333ba117c68d49486dfd7ff41
> X-Git-Archeology:   Date: Sun, 20 Mar 2022 22:58:21 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: Adding Pine64 Quartz64a as WIP target (#3539)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0afe24c95729044910e0b3f84dc5500bcdc6524c: https://github.com/armbian/build/commit/0afe24c95729044910e0b3f84dc5500bcdc6524c
> X-Git-Archeology:   Date: Sun, 24 Apr 2022 22:33:47 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media edge to 5.17 (#3704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c: https://github.com/armbian/build/commit/2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c
> X-Git-Archeology:   Date: Mon, 04 Jul 2022 16:18:31 +0300
> X-Git-Archeology:   From: balbes150 <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 5.18 and edge 5.19
> X-Git-Archeology:
> X-Git-Archeology: - Revision 738c1613073bfd504aebc223a0d110cc6afefbe5: https://github.com/armbian/build/commit/738c1613073bfd504aebc223a0d110cc6afefbe5
> X-Git-Archeology:   Date: Sat, 12 Nov 2022 10:11:22 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media-current to 5.19.17 (#4406)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 276f346ee9c21cdcd627e54676334f4d5a9292b6: https://github.com/armbian/build/commit/276f346ee9c21cdcd627e54676334f4d5a9292b6
> X-Git-Archeology:   Date: Wed, 14 Dec 2022 10:17:09 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 6.0 and edge 6.1 (#4566)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 16 +++++-----
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index d4a65294ba2a..684cdcc0a4b5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -2528,25 +2528,25 @@ sdio0_int: sdio0-int {
 		sdmmc {
 			sdmmc_bus1: sdmmc-bus1 {
 				rockchip,pins =
-					<4 RK_PB0 1 &pcfg_pull_up>;
+					<4 RK_PB0 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_bus4: sdmmc-bus4 {
 				rockchip,pins =
-					<4 RK_PB0 1 &pcfg_pull_up>,
-					<4 RK_PB1 1 &pcfg_pull_up>,
-					<4 RK_PB2 1 &pcfg_pull_up>,
-					<4 RK_PB3 1 &pcfg_pull_up>;
+					<4 RK_PB0 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB1 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB2 1 &pcfg_pull_up_8ma>,
+					<4 RK_PB3 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_clk: sdmmc-clk {
 				rockchip,pins =
-					<4 RK_PB4 1 &pcfg_pull_none>;
+					<4 RK_PB4 1 &pcfg_pull_none_12ma>;
 			};
 
 			sdmmc_cmd: sdmmc-cmd {
 				rockchip,pins =
-					<4 RK_PB5 1 &pcfg_pull_up>;
+					<4 RK_PB5 1 &pcfg_pull_up_8ma>;
 			};
 
 			sdmmc_cd: sdmmc-cd {
@@ -2556,7 +2556,7 @@ sdmmc_cd: sdmmc-cd {
 
 			sdmmc_wp: sdmmc-wp {
 				rockchip,pins =
-					<0 RK_PB0 1 &pcfg_pull_up>;
+					<0 RK_PB0 1 &pcfg_pull_up_8ma>;
 			};
 		};
 
-- 
Armbian

